features/supplier/view_and_edit_services.feature

```diff
@@ -24,3 +25,22 @@ Scenario Outline: Supplier coming from dashboard to view the detail page for one
     | digital-outcomes           | Digital outcomes           | Team capabilities           | Agile coaching   |
     | user-research-participants | User research participants | Recruitment approach        | Entirely offline |
     | user-research-studios      | GDSvieux Innovation Lab    | Lab address                 | GDSbury          |
+
+@skip-preview
```

Comment:
Be nice to throw in `@skip-local` as well, for anyone who tries to run the functional tests locally.